packages/components/src/shortcut/test/index.tsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import { render, screen } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import Shortcut from '..';
+
+describe( 'Shortcut', () => {
+	it( 'does not render anything if no shortcut prop is provided', () => {
+		const { container } = render( <Shortcut /> );
+		expect( container.firstChild ).toBe( null );
+	} );
+
+	it( 'renders the shortcut display text when a string is passed as the shortcut', () => {
+		render( <Shortcut shortcut="shortcut text" /> );
+		expect( screen.getByText( 'shortcut text' ) ).toMatchSnapshot();
+	} );
+
+	it( 'renders the shortcut display text and aria-label when an object is passed as the shortcut with the correct properties', () => {
+		render(
+			<Shortcut
+				shortcut={ {
+					display: 'shortcut text',
+					ariaLabel: 'shortcut label',
+				} }
+			/>
+		);
+
+		expect( screen.getByText( 'shortcut text' ) ).toMatchSnapshot();
```

Comment:
I wonder if we could write this test in a way that explicitly tests the accessible label, maybe something like:

```suggestion
		expect( screen. getByLabelText( 'shortcut label' ) ).toHaveTextContent('shortcut text');
```

Reply:
Yep. Will take a look at it 👍

Reply:
https://github.com/WordPress/gutenberg/pull/42272/commits/041ee3b952f4a6be92ecd3cae8b90ae142e6de40 looks good!

In general, my preference would be to prefer this style of tests over snapshots when possible.

Thank you, and feel free to merge!